### ISSUE TITLE
Failed to load PostCSS config 

### ISSUE THREAD

**Issue by coder36** [TiagoSilvaPereira/vemto-issues#431]
Attention: Please use the Discussions page for feature requests or suggestions instead of opening a new issue.

**Describe the bug**
Unable to generate code.  Stops with:
```
[vite:css] Failed to load PostCSS config (searchPath: /Users/mark/dev/parish-admin/parish_admin): [Failed to load PostCSS config] Failed to load PostCSS config (searchPath: /Users/mark/dev/parish-admin/parish_admin): [ReferenceError] module is not defined in ES module scope
This file is being treated as an ES module because it has a '.js' file extension and '/Users/mark/dev/parish-admin/parish_admin/package.json' contains "type": "module". To treat it as a CommonJS script, rename it to use the '.cjs' file extension.
ReferenceError: module is not defined in ES module scope
This file is being treated as an ES module because it has a '.js' file extension and '/Users/mark/dev/parish-admin/parish_admin/package.json' contains "type": "module". To treat it as a CommonJS script, rename it to use the '.cjs' file extension.
```

**To Reproduce**
Create a vanilla project

**Expected behaviour**
It should not fail

**Screenshots**

<img width="1414" alt="image" src="https://github.com/TiagoSilvaPereira/vemto-issues/assets/561451/0eac3b5b-9010-4492-b30f-4db80a572907">

**Desktop (please complete the following information):**
 - OS: IOS
 - Browser Chrome
 - node: 18.16.0

**Comment by coder36** [comment]
I can get it to work by editing package.json and removing `"type": "module"

**Comment by TiagoSilvaPereira** [comment]
Hi @coder36 it is happening due to recent changes on Vite. The correct way to solve it is by following the error message: rename postcss.config.js to postcss.config.cjs... I'm looking for a way to solve it without workarounds or changing the default Laravel skeleton

**Comment by TiagoSilvaPereira** [comment]
@coder36 fixed on Vemto 1.3.3
